UPGRADE-2.0.md

```diff
@@ -0,0 +1,6 @@
+# Upgrade guide from 1.x to 2.0
+
+## Needed actions
+This is the list of actions that you need to take when upgrading this bundle from the 1.x to the 2.0 version:
+
+ * ...
```

Comment:
It may consist of something like that:

```
composer remove --dev `nelmio/alice`
composer require --dev liip/functional-test-bundle:~2.0
```

Reply:
@Jean85 Please note that I wrote this as an example, I didn't tested it. :wink: 

And I used backticks `, composer accepts them but they should probably be removed in order to prevent potential errors.

Reply:
Roger! Fixed accordingly